prov/kubernetes/monitoring_job.go

```diff
@@ -53,8 +53,8 @@ func (o *actionOperator) ExecAction(ctx context.Context, cfg config.Configuratio
 	}
 	// Check if namespace was provided
 	var namespaceProvided bool
-	namespaceProvidedStr, ok := action.Data["providedNamespace"]
-	if b, err := strconv.ParseBool(namespaceProvidedStr); ok && err != nil {
+	namespaceProvidedStr, ok := action.Data["namespaceProvided"]
+	if b, err := strconv.ParseBool(namespaceProvidedStr); ok && err == nil {
```

Comment:
Maybe we should check first ok is true before converting namespaceProvidedStr to bool ?

Reply:
Yes in fact. Change done